src/utilities/NewsApi.js

```diff
@@ -7,7 +7,7 @@ class Api {
   }
 
   _customFetch(url, options) {
-    return fetch(url, options).then((res) => (res.ok ? res.json() : Promise.reject(res.statusText)));
+    return fetch(url, options).then((res) => (res.ok ? res : Promise.reject(res.statusText)));
```

Comment:
`res.json()` should not be duplicated in your request in `App`. Please, use it only in `_customFetch` or `_checkResponse` otherwise you'll get bugs or duplication of the logics